css-index.css

```diff
@@ -0,0 +1,76 @@
+@media (min-width:900px){
+  .container{
+    width: auto;
+    border: 3px solid black;
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    grid-auto-rows: 100px;
+  }
+  .container > div{
+    border: 1px solid black;
+  }
+  .navegation{
+    display: flex;
+    align-self: flex-end;
+    grid-column: 1/5;
+  }
+  .name{
+    display: flex;
+    color: azure;
+    flex-grow: 4;
+    font-size: 60px;
+  }
+  .buttons{
+    display:flex;
+    padding-bottom: 20px;
+    justify-content: flex-end;
+  }
+  button{
+    color: aliceblue;
+    background-color: black;
+    border-radius: 30px;
+  }
+
+}
+@media (max-width:899px){
+  .container{
+    width: auto;
+    border: 3px solid black;
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+    grid-auto-rows: 100px;
+  }
+  .container > div{
+    border: 1px solid white;
+  }
+  .navegation{
+    display: flex;
+    align-self: flex-end;
+    grid-column: 1/4;
+  }
+  .name{
+    display: flex;
+    color: azure;
+    flex-grow: 4;
+    align-self: center;
+    font-size: 60px;
+  }
+  .buttons{
+    display: flex;
+    flex-direction: column;
+  }
+  button{
+    color: aliceblue;
+    background-color: black;
+    border-radius: 30px;
+    display:flex;
+    padding-bottom: 10px;
+    justify-content: flex-end;
+  }
+  
+}
+@media (max-width:745px){
```

Comment:
good job, you have used the medias correctly, nice :D

Reply:
thank you